ParkenDD/SettingsView/SettingsViewController.swift

```diff
@@ -275,6 +275,10 @@ class SettingsViewController: UITableViewController, MFMailComposeViewController
 				break
 			}
 		}
+		
+		if let lotlistVC = UIApplication.shared.keyWindow?.rootViewController?.childViewControllers[0] as? LotlistViewController {
```

Comment:
You're tightly coupling Settings with `LotListViewcontroller`. In theory, Settings should never have a direct reference to `LotListViewController`. I'm pretty sure you don't need this as you can easily call `updateData` on `LotListViewController`'s `viewWillAppear` func.

Reply:
I get it but this code was original code from project - not mine - and I left it 'AS IS' in order to not refactor much code.

Reply:
updateData is called on LotListViewController too so I will eliminate the above code.

Reply:
It wont automatically refresh data if you don't call refreshLotlist() and you would have to manually pull to refresh then.